swagger/swagger.yaml

```diff
@@ -1461,12 +1473,14 @@ components:
         - maroon
     AidTypeEnum:
       type: string
-      description: Means of evacuation
+      description: Type of aid
```

Comment:
Should be `types`

Reply:
I had thought that at first too, but i think it's just one single evac type

Reply:
I don't have a problem changing to "types".  The enum specifies the different types of aid (local military aid, local non-military aid, ground evac, etc.).